db/migrate/20211103090940_translate_budget_main_link_url.rb

```diff
@@ -0,0 +1,6 @@
+class TranslateBudgetMainLinkUrl < ActiveRecord::Migration[5.2]
+  def change
+    add_column :budget_translations, :main_link_url, :string
+    remove_column :budgets, :main_link_url
```

Comment:
```suggestion
    remove_column :budgets, :main_link_url, :string
```

This way we make the migration reversible :wink:.

Reply:
Done!